features/built_in_matchers/include.feature

```diff
@@ -1,7 +1,8 @@
 Feature: `include` matcher
 
-  Use the `include` matcher to specify that a collection includes one or more expected objects.
-  This works on any object that responds to `#include?` (such as a string or array):
+  Use the `include` matcher to specify that a collection includes one or more
+  expected objects. You can also think of it as an `any`-matcher. This works on
+  any object that responds to `#include?` (such as a string or array):
```

Comment:
just my two penneth here, but i'm not sure how i feel about thinking of `include` as an `any`-matcher, since we can do this with enumerables:

```
expect([]).not_to be_any
```

(though it might not make complete sense). i guess i just find thinking of a matcher in terms of some other matcher a little confusing.